tests/Agent/UnitTests/Core.UnitTest/NewRelic.Agent.Core.FromLegacy/AgentLoggerTest.cs

```diff
@@ -20,21 +15,28 @@ namespace NewRelic.Agent.Core
     public class LoggerBootstrapperTest
     {
         [Test]
-        public static void IsDebugEnabled_is_false_when_config_log_is_off()
+        public static void no_log_levels_are_enabled_when_config_log_is_off()
```

Comment:
I think we should capitalize the start of the test names. It looks a bit odd without it.